nle/scripts/plot.py

```diff
@@ -88,13 +99,17 @@ def str_to_float_pair(s):
     "--xrange",
     type="pair",
     default=None,
-    help="float,float. range of x values to plot. overrides automatic zoom for x axis.",
+    help="""
```

Comment:
No need to change this line I think?

Reply:
And if there's need to cut down the line length, let's make it so that the result stays the same. One option would be

```
help=("..."
           "..."),
```

Reply:
yeah I can fix this - this is a personal pep8 setting